packages/amplify-frontend-flutter/lib/configuration-manager.js

```diff
@@ -30,6 +30,21 @@ function normalizeInputParams(context) {
   context.exeInfo.inputParams[constants.Label] = inputParams;
 }
 
+export function displayFrontendDefaults(context) {
```

Comment:
Hmm. I thought it made more sense to allow the plugin to display the data since each set of info is specific to the plugin - similar to the way we let the plugins ask the frontend specific questions.